dooble/marble.py

```diff
@@ -117,11 +117,8 @@ def emission_links(top_layer, bottom_layer, items):
                 items = []
                 top_layer = layer_index
             elif type(layer) is Observable:
-                if layer.label is not None:
```

Comment:
The original code is the expected behavior. However as explained on the comments, we can render the arrow differently. This would mean computed to lists here: one for items links and one for label links.